data.json

```diff
@@ -449,6 +449,15 @@
     "dateAdded": "2019-10-10T13:00:00.000Z",
     "tags": ["hacktoberfest", "clothing", "expired"]
   },
+  {
+    "name": "GoFr",
+    "difficulty": "medium",
+    "description": "Contribute to GoFr by submitting a notable pull request or by writing an article. contributions in someway to development or spreading the word about GoFr. receive GoFr T-Shirt and Stickers as a token of appreciation.",
+    "reference": "https://github.com/gofr-dev/gofr",
+    "image": "https://gofr.dev/_next/static/media/complete-gorg-logo.bc9195c4.svg",
```

Comment:
Please use some other link, these next assets are likely to change path at every build.

Reply:
for image URL?